tests/integration/test_contract.py

```diff
@@ -177,12 +177,12 @@ class TestContractRestAPI(TestContract):
 
     def _get_network_config(self):
```

Comment:
In some integration tests we deploy sample contracts on testnet in every run! This leads to adding unnecessary data there.